test/zdtm.py

```diff
@@ -2604,6 +2620,9 @@ def clean_stuff(opts):
                 help="Use splice or read mode of pre-dumping",
                 choices=['splice', 'read'],
                 default='splice')
+rp.add_argument("--non-root",
```

Comment:
we already have the --user option. How is this one different?

Reply:
Not at all. I was not aware of `--user`. I have updated this PR to use `--user` instead of `--non-root`.

Reply:
Can we rename --user to --non-root? user could mean anything. (criU, user namespaces). Not at all obvious of what it would do. --unprivileged is pretty excplicit too. I'd argue that Adrian might have seen the option if the naming of the option was unambiguous. Our users might miss it too.

Reply:
What is the history behind `--user`? I see now that is has been used in one of the jenkins jobs which I am not familiar with.

Is `--user` already testing exactly the same I wanted to test with this PR? Running CRIU with CAP_CHECKPOINT_RESTORE/CAP_SYS_ADMIN?

Reply:
> What is the history behind --user?

It was Pavel's attempt to implement criu dump that can't be executed by non-root users to dump their processes.

Reply:
Can I suggest `--rootless`? It is an established term and the meaning is very clear.

Reply:
My first patch series was using `--non-root`, now I am currently using `--user`. `--rootless` is indeed a good idea, but I do not really care which name it is.

Should we re-use `--user` or give it a new name?

Reply:
The idea of the --user option is to allow non-root users dumping processes even when criu doesn't have any capabilities. We want to save this functionality.

Reply:
Renamed my changes to `--rootless`.